kyuubi-server/src/test/scala/org/apache/kyuubi/server/KyuubiRestFrontendServiceSuite.scala

```diff
@@ -17,80 +17,39 @@
 
 package org.apache.kyuubi.server
 
-import java.util.Locale
+import org.apache.kyuubi.RestFrontendTestHelper
 
-import scala.io.Source
-
-import org.scalatest.time.SpanSugar._
-
-import org.apache.kyuubi.{KyuubiFunSuite, RestFrontendTestHelper}
-import org.apache.kyuubi.config.KyuubiConf
-import org.apache.kyuubi.service.NoopRestFrontendServer
-import org.apache.kyuubi.service.ServiceState._
-
-class KyuubiRestFrontendServiceSuite extends KyuubiFunSuite with RestFrontendTestHelper {
-
-  test("kyuubi REST frontend service basic") {
```

Comment:
this case is not very necessary as most logic will be verified in RestFrontendTestHelper